config/.env.example

```diff
@@ -1,18 +1,20 @@
-NODE_ENV=local
-PORT=3000
-API_ROOT_URL=http://localhost:3000
-FRONT_BASE_URL=http://localhost:4200
+# Secrets
+# YOU MUST CHANGE THESE BEFORE GOING INTO PRODUCTION
+JWT_SECRET=your-secret
 
-MONGO_URL=mongodb://localhost:27017/notifire-db
-REDIS_PORT=6379
-REDIS_HOST=localhost
+## General
+NODE_ENV=test
+MONGO_URL=mongodb://mongodb:27017/notifire-db
+REDIS_HOST=redis
 REDIS_DB_INDEX=2
+NEW_RELIC_ENABLED=false
```

Comment:
Can we remove it from the .env.example and just add this as a static env in `docker-compose.yml` ? I want to reduce as much mental overhead as possible from this file.

Reply:
Agree i will try to make this folder as lean as possible.